toytools/transform.py

```diff
@@ -0,0 +1,74 @@
+"""Image transformation tools"""
+
+from typing import Tuple
+import numpy as np
+
+def get_background_value(image):
+    """Return inferred background value of image"""
+    values, counts = np.unique(image, return_counts = True)
+    idx_max = np.argmax(counts)
+    return values[idx_max]
+
+def get_background_value_fast(image):
+    """Return fast but possible incorrect inferred background value of image"""
+    return get_background_value(image[:128, :128])
```

Comment:
Best to avoid magic numbers in code.  At least pull them out to args like

```python
def get_background_value_fast(image, shape=(128,128)):
```

Reply:
Thank you! I have fixed that.